packages/bitcore-wallet-client/src/lib/api.ts

```diff
@@ -2116,6 +2129,36 @@ export class API extends EventEmitter {
     });
   }
 
+  // /**
+  // * Returns contract instantiation info.
+  // * @param {string} opts.sender - sender eth wallet address
+  // * @param {string} opts.network - the network for this wallet
+  // * @param {string} opts.coin - the coin for this wallet
+  // * @return {Callback} cb - Return error (if exists) instantiation info
+  // */
+  getMultisigContractInstantiationInfo(opts, cb) {
+    var url = '/v1/ethmultisig/';
+    this.request.post(url, opts, (err, contractInstantiationInfo) => {
+      if (err) return cb(err);
+      return cb(null, contractInstantiationInfo);
+    });
+  }
+
+  // /**
+  // * Returns contract info.
```

Comment:
please provide more details about the information that will be returned.

Reply:
Done. Thanks